client/src/ui/LandingNavBar.tsx

```diff
@@ -0,0 +1,144 @@
+"use client";
+import Image from "next/image";
+import Link from "next/link";
+import { Button, Typography } from "@mui/material";
+import { customTheme } from "@/theme/theme";
+import { ThemeProvider } from "@mui/material";
+
+
+/*
+ * This Landing Navbar will be responsible for rendering the Navbar of the Landing Page
+ */
+export function LandingNavbar() {
+  return (
+    <nav
+      style={{
+        background: "rgb(255, 255, 255)",
```

Comment:
nit: Just do "white" to make it more obvious what the color is"

Reply:
It has been fixed!